src/containers/org/members/FilteredRoles.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import React from 'react';
+
+import { Checkbox } from 'lattice-ui-kit';
+import type { Role, UUID } from 'lattice';
+
+type Props = {
+  filterTerm :string;
+  onChange :(e :SyntheticEvent<HTMLInputElement>) => void;
+  roles :Role[];
+  selectedRoleId :UUID | null;
+};
+
+const FilteredRoles = ({
+  filterTerm,
+  onChange,
+  roles,
+  selectedRoleId,
+} :Props) => {
+
+  let filteredRoles = roles;
+  if (filterTerm) {
+    filteredRoles = filteredRoles.filter((role) => role.title.toLowerCase().includes(filterTerm));
```

Comment:
This bug got me in the orgs filter... you'll want to `.toLowerCase()` the filter term as well.